.NET/Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishDateTimePeriodParserConfiguration.cs

```diff
@@ -204,5 +212,45 @@ public int GetSwiftPrefix(string text)
 
             return swift;
         }
+
+        private bool GetMatchedTimeRangeForTasksMode(string text, out string todSymbol, out int beginHour, out int endHour, out int endMin)
```

Comment:
Could this be a static method in the task-specific utils file?

Reply:
shifted code block to tasksmode specific utils.